corehq/apps/domain/forms.py

```diff
@@ -776,6 +776,8 @@ def __init__(self, can_edit_eula, *args, **kwargs):
 
         self.helper = FormHelper()
         self.helper.form_class = 'form form-horizontal'
+        self.helper.label_class = 'col-lg-2'
+        self.helper.field_class = 'col-lg-10'
```

Comment:
this should be `col-sm-8`---I've found that completely spanning the page is not great for a lot of the fields, and this has been a good starting size.